client/public/config/configuration.json

```diff
@@ -1,12 +1,21 @@
 [
   {
-    "URL":"https://namex-dev.pathfinder.gov.bc.ca",
+    "URL": "https://namex-dev.pathfinder.gov.bc.ca",
     "PAYMENT_PORTAL_URL": "https://dev.bcregistry.ca/business/auth/makepayment",
     "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1",
     "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
     "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
     "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
     "LEGAL_API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1",
-    "LD_CLIENT_ID": ""
+    "LD_CLIENT_ID": "",
+    "NR_COMPLETION_TIMER_NAME": "nrCompletionTimer",
+    "NR_COMPLETION_TIMEOUT_MS": 0,
+    "DEV_NR_COMPLETION_TIMEOUT_MS": 0,
+    "EXISTING_NR_TIMER_NAME": "existingNrTimer",
+    "EXISTING_NR_TIMEOUT_MS": 0,
+    "DEV_EXISTING_NR_TIMEOUT_MS": 0,
+    "PAYMENT_COMPLETION_TIMER_NAME": "paymentCompletionTimer",
+    "PAYMENT_COMPLETION_TIMEOUT_MS": 120000,
+    "DEV_PAYMENT_COMPLETION_TIMEOUT_MS": 120000
```

Comment:
Please add check items to the NE Release ticket (/bcgov/entity#5732) to add these to the Dev/Test/Prod config maps for deployment.

Reply:
@bcdevlucas Please confirm ^^.

Reply:
Noted.

Reply:
@bcdevlucas Please add check items to the NR Release ticket.

(You may not be around when we deploy and we may not remember to do this.)